app/controllers/admin/invoices_controller.rb

```diff
@@ -0,0 +1,83 @@
+class Admin
```

Comment:
Consider aggregating invoices across partners in the org 🍊

This snippet might be useful
```ruby
        membership = Membership.find_by(user_id: current_publisher.id)
        organization = Organization.find(membership.organization_id)
        partners = Membership.where(organization_id: organization.id).map { |membership| membership.user_id}
```

Reply:
So my only issue is that if I do that we'd need to display all the invoices on the organization page. Which is doable, but since each uphold account is linked to a user then we'd need to figure out which account is going to be the one that gets paid out. 

I think we can go back and iterate on this once we have some actual usage information. For now we should communicate with BizDev that it's one partner per account?

Reply:
Isn't that also true as it stands right now? Any partner in an organization can upload invoices, so bd will need to decide which partner from the organization to payout.

Reply:
So the way that the system currently works is the invoice button displays the most recent invoice available to the partner, and this is per partner. So I'd have to figure out some way to display multiple invoices on the front end for the different partners. One solution is sharing the invoice across the organization

This would allow different users to login and upload their own reports to that one invoice - this is totally doable. Given Mandar's statement on how invoices and reports come from different areas of the same company it might make sense to have a couple of different accounts which can access the same invoice. Otherwise it's kind of a useless functionality 🤔 

I'd be worried about having to redo some the UI that we have already demo'd to make this possible, but I'm optionally for it if this is something that we thinks makes more sense

Curious on what your thoughts @yachtcaptain23